src/__tests__/dbOperationAuditLogs.int.test.ts

```diff
@@ -0,0 +1,40 @@
+import { describe, expect, test } from '@jest/globals';
```

Comment:
Why are we adopting a different pattern here?  We don't import these in any of our other tests!

Reply:
Let me see what errors occur if I remove these. For whatever reason, codium was not happy without this.

Reply:
Weird. No bad thing occurs when I remove it and the associated dependency. :shrug: Thanks!

Reply:
I limited the import to `expect` in #1630 and made a comment explaining why it is there.